src/rust/devcontainer-feature.json

```diff
@@ -10,6 +10,9 @@
             "proposals": [
                 "latest",
                 "none",
+                "1.79",
```

Comment:
Can you help bump patch version of this Feature? thanks! 

https://github.com/devcontainers/features/blob/a7c5141c6888d90d5fa1daa33b217f37cee39b36/src/rust/devcontainer-feature.json#L3

Reply:
Updated. Could you have a look again. Thanks in advance!